internal/rbd/encryption.go

```diff
@@ -111,6 +111,41 @@ func (ri *rbdImage) setupEncryption(ctx context.Context) error {
 	return nil
 }
 
+func (ri *rbdImage) copyEncryptionConfig(cp *rbdImage) error {
+	// get the unencrypted passphrase
```

Comment:
Isnt this we are getting `crypt` passphrase . Got confused by above comment saying `unencrypted` passphrase

Reply:
yes, `unencrypted` is correct. Passphrases can be encrypted per volume, so a copy of the passphrase for the next volume will get (potentially) encrypted differently. The unencrypted passphrase will remain the same. I hope the comments inside the function make this process clear.

A new comment describing the function will also clarify this more.